LICENSES/LicenseRef-elixir-trademark-policy.txt

```diff
@@ -0,0 +1 @@
+For terms of use and license, see https://elixir-lang.org/trademarks
```

Comment:
This license could be added to the scancode licensedb for future detection :)

An example of where to submit follows :) 
https://github.com/aboutcode-org/scancode-licensedb/blob/main/docs/adaptec-downloadable.json

Reply:
By chance I'm standing next to a maintainer from the ScanCode team at a CRA Workshop of the Eclipse Foundation. Apparently the right way is to just open a GitHub issue on the LicenseDB.

I'll look into that.

Reply:
Let me know once we are ready to go forward then!

Reply:
Sounds good to me. Shall we merge this then? Any other blocker from your side?

Reply:
Reported as https://github.com/aboutcode-org/scancode-licensedb/issues/66